src/codegen/mod.rs

```diff
@@ -1012,10 +1020,12 @@ impl CodeGenerator for CompInfo {
 
             // Try to catch a bitfield contination early.
             if let (Some(ref mut bitfield_width), Some(width)) =
-                   (current_bitfield_width, field.bitfield()) {
+                (current_bitfield_width, field.bitfield()) {
```

Comment:
I often wouldn't make the specific choices `rustfmt` makes, but that's not really the point ;) The point is not having to make the choices, and argue about them with each other.